uber/templates/static_views/js/prereg-common.js

```diff
@@ -0,0 +1,183 @@
+var PREREG_CHECK_INTERVAL = 15 * 60 * 1000;
+var checkCap = function () {
+    $.getJSON('../preregistration/check_prereg', function (check) {
+        if (check.force_refresh) {
+            location.reload();  // Reload the page to prevent registering after reg is closed
+        } else {
+            setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+        }
+    });
+};
+setTimeout(checkCap, PREREG_CHECK_INTERVAL);
+
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
Oooh, I was actually getting a bit confused. You had said that we can't use context-aware data like an `attendee` object, and I took that to mean that we couldn't use django-style tags at all.